omod/src/main/webapp/pages/registrationSummary.gsp

```diff
@@ -32,4 +32,4 @@
 
 ${ ui.includeFragment("coreapps", "patientHeader", [ patient: patient.patient, activeVisit: null, appContextModel: null ]) }
 
-${ ui.includeFragment("registrationapp", "summary/registrationSummary", [patient: patient, appId: "registrationapp.registerPatient"]) }
\ No newline at end of file
+${ ui.includeFragment("registrationapp", "summary/registrationSummary", [patient: patient, appId: appId]) }
```

Comment:
@dkayiwa https://github.com/openmrs/openmrs-module-registrationapp/blob/master/omod/src/main/java/org/openmrs/module/registrationapp/page/controller/RegistrationSummaryPageController.java#L50 

However it seems like there is no default value, which I am adding in a followup commit, which I assume was the cause of the hard-coding